datadotworld/client/api.py

```diff
@@ -143,14 +143,6 @@ def create_dataset(self, owner_id, **kwargs):
             lambda: _swagger.DatasetCreateRequest(
                 title=kwargs.get('title'),
                 visibility=kwargs.get('visibility')),
-            lambda name, url, expand_archive, description, labels:
```

Comment:
I think you cab still add files on dataset create ? But only via url.

Reply:
I had removed it after seeing [this comment](https://github.com/datadotworld/data.world-py/pull/132#discussion_r981554171); I can put it back and use the dummy lambdas in `update_dataset` and `replace_dataset`

Reply:
I don't think dummy lambdas are a good idea in any case. You probably want method overloading. But what you ultimately want is the python SDK to exactly reflect the Public API spec.